RecoJets/JetProducers/python/ECF_cfi.py

```diff
@@ -2,6 +2,90 @@
 
 ECF = cms.EDProducer("ECFAdder",
              src = cms.InputTag("ak8CHSJets"),
+             ecftype = cms.string("ECF"),
              Njets = cms.vuint32(1, 2, 3),
+             alpha = cms.double(1.0),        # CMS default is 1
              beta = cms.double(1.0),        # CMS default is 1
              )
+
+ECFCbeta1 = cms.EDProducer("ECFAdder",
```

Comment:
In principle, for these modules the same CMS capitalization rules used for the class members in c++ code, i.e. (e.g.)
ECFCbeta1 -> ecfCbeta1
(and the same for the other modules defined here)